core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/job/StructuredSynchronizerJobAdapter.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.structuredevent.job;
+
+import org.quartz.Job;
+import org.springframework.context.ApplicationContext;
+import org.springframework.data.repository.CrudRepository;
+
+import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.quartz.model.JobResourceAdapter;
+import com.sequenceiq.cloudbreak.repository.StackRepository;
+
+public class StructuredSynchronizerJobAdapter extends JobResourceAdapter<Stack> {
```

Comment:
Do we need the full Stack here? Maybe using StackView or similar would consume less resources.
Btw, have we got any estimate about the load this new scheduled task would generate?